docker/README.md

```diff
@@ -84,7 +81,7 @@ If you do not see your problem captured, please file a [new issue][] and follow
 
 ## Legal and Licensing
 
-Azure-PowerShell is licensed under the [Apache license][].
+AzureStack PowerShell is licensed under the [Apache license][].
 
 [Apache license]: https://github.com/Azure/azure-powershell/blob/master/LICENSE.txt
 
```

Comment:
Pls add instructions on how to run the tests, with required parameters and their description